### PR TITLE
DSND-2688: Fix failing transformations

### PR DESCRIPTION
**Pull request by sthompsonCH**
## Describe the changes
This PR fixes failing transformations when transforming LLPSC07, PSC07, SLPPSC07, and SQPPSC07 when their deltas don't have a description values object.

### Related Jira tickets
[DSND-2688](https://companieshouse.atlassian.net/browse/DSND-2688)

### General
- [ ] Is the PR as small as it can be?
- [ ] Has the Jira ticket been updated with any relevant comments?
- [ ] Is the `README` up to date?
- [ ] Has the `POM` been updated for the latest versions of dependencies?
- [ ] Has the code been double-checked against `main`?
- [ ] Does the code adhere standards in this repository, including SonarQube checks?

### Testing
- [ ] Do the code changes have unit and integration tests with code coverage > 80%?
- [ ] Has the code been tested locally and/or passed the API Karate tests on Tilt?
- [ ] Have mandatory manual test cases been run?
- [ ] Are extra Karate tests required?
- [ ] Are all the test data resources up to date?

### Release preparation
_Where possible, add links to the respective Jira tickets when an item is checked_
- [ ] Have these changes been included in a release ticket?
- [ ] Are changes required to the `docker-chs-development` deployment or test data?
- [ ] Are any changes required to the environment added to `chs-configs`?

## Notes to the tester
_Add any testing hints or instructions here._

[DSND-2688]: https://companieshouse.atlassian.net/browse/DSND-2688?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/test/resources/data/mortgage/SLPPSC07_delta.json [hunk 1]
```diff
@@ -2,18 +2,18 @@
...
       "parent_entity_id": "",
       "parent_form_type": "",
       "description_values": {
-        "cessation_date": "24/08/2016",
-        "psc_name": "JOHN TESTER"
+        "cessation_date": "14/08/2017",
+        "psc_name": "JOHN TESTER GP LLP"
       },
       "pre_scanned_batch": "1"
     }
```
[the remark this answers is not in the document]
Comment (sthompsonCH) [reply src/test/resources/data/mortgage/SLPPSC07_delta.json]: > Is this missing a `delta_at`?

No, it's on line 21

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by johncook-ch — APPROVED** [review]

**Pull request merged**